gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/exception/HttpListenerPathMissingException.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright © 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.apim.core.exception;
+
+public class HttpListenerPathMissingException extends RuntimeException {
```

Comment:
What about renaming it to `HttpListenerPathInvalidException`? Also instead of throwing `InvalidHostException` and `PathAlreadyExists` (not completely sure of the last one) we could throw this `HttpListenerPathInvalidException` 🤔 

What about using a regular `Exception`? It would make it explicit that the user of the domain service needs to deal with the situation when the paths list provided is incorrect?